README.md

```diff
@@ -1 +1,135 @@
-# hello-back-end-rails
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+</div>
+
+
+> # Rails Recipe
+
```

Comment:
- [ ] Please ensure to update the project title both here and in the front-end readme file. :ok_hand: